packages/create-cloudflare/src/__tests__/dialog.test.ts

```diff
@@ -0,0 +1,137 @@
+				 │ 🎉  SUCCESS  Application deployed successfully!                                       │
+				 │                                                                                       │
+				 │ 🔍 View Project                                                                       │
+				 │    Visit: https://example.test.workers.dev                                            │
+				 │    Dash: https://dash.cloudflare.com/?to=/:account/workers/services/view/test-project │
+				 │                                                                                       │
+				 │ 💻 Continue Developing                                                                │
+				 │    Start dev server: npm run start                                                   │
+				 │    Deploy again: npm run deploy                                                      │
+				 │                                                                                       │
+				 │ 📖 Explore Documentation                                                              │
+				 │    https://developers.cloudflare.com/workers                                          │
+				 │                                                                                       │
+				 │ 💬 Join our Community                                                                 │
+				 │    https://discord.cloudflare.com                                                     │
+				 ╰───────────────────────────────────────────────────────────────────────────────────────╯
+				"
+			`);
+		});
+
+		test("with no deploy", async () => {
+			await printSummary({
+				...ctx,
+				account: undefined,
+				deployment: {},
+				project: { name: "test-project", path: "./example" },
+				template: {
+					...ctx.template,
+					platform: "pages",
+				},
+			});
+
+			expect(normalizeOutput(std.out)).toMatchInlineSnapshot(`
+				" ╭──────────────────────────────────────────────────────────────╮
+				 │ 🎉  SUCCESS  Application created successfully!               │
+				 │                                                              │
+				 │ 💻 Continue Developing                                       │
+				 │    Change directories: cd ../example                         │
+				 │    Start dev server: npm run start                          │
+				 │    Deploy: npm run deploy                                   │
+				 │                                                              │
+				 │ 📖 Explore Documentation                                     │
+				 │    https://developers.cloudflare.com/pages                   │
+				 │                                                              │
+				 │ 💬 Join our Community                                        │
+				 │    https://discord.cloudflare.com                            │
+				 ╰──────────────────────────────────────────────────────────────╯
+				"
+			`);
+		});
+
+		test("with lines truncated", async () => {
+			process.stdout.columns = 40;
```

Comment:
This should be unset in an afterEach to not affect any following tests (there aren't any now but additional tests may be added in the future and it shouldn't be on the next dev to figure that out)